File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/non-plagiarized/01/T03.py
import sys

def main():
    weight = int(input("Enter weight in pounds: "))
    feet = int(input("Enter feet: "))
    inches = int(input("Enter inches: "))

    height = feet * 12 + inches
    bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

    print(f"BMI is {bmi:.2f}")

    if bmi < 18.5:
        print("Underweight")
    elif 18.5 <= bmi < 25:
        print("Normal")
    elif bmi >= 35:
        print("Obese")

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/non-plagiarized/09/t03.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter weight in pounds : ', (weightStr) => {
  const weight = parseFloat(weightStr);

  rl.question('Enter feet : ', (feetStr) => {
    const feet = parseFloat(feetStr);

    rl.question('Enter inches : ', (inchesStr) => {
      const inches = parseFloat(inchesStr);

      const height = feet * 12 + inches;
      const bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
      console.log('BMI is ' + bmi);

      if (bmi < 18.5) {
        console.log('underweight');
      } else if (bmi >= 18.5 && bmi < 25) {
        console.log('normal');
      } else if (bmi >= 25 && bmi < 35) {
        console.log('overweight');
      } else {
        console.log('obese');
      }

      rl.close();
    });
  });
});

